### DISCUSSION TITLE
Trinity Firefly migration, searching my iota

### DISCUSSION THREAD

**Discussion by jklm2828** [iotaledger/firefly#7753] [Help & Support]
Hello,

a few hours ago I tried to log into my Trinity account. I wanted to hodl my iota for a long time thats why i haven't checked it for years... I figured out that Trinity is offline and a new wallet named Firefly should be used. I want to access my iota.
I downloaded the Firefly software from the linked website I found in the Trinity programm. I tried to recover my iota profile. I have no ledger device but I have my recovery words from the Trinity wallet and the restored file (kdbx file).
The recovery words from Trinity are 27, so i can't use them. And the "Use Stronghold backup" hasn't worked. I selected the file but it somehow doesn't upload it properly because I couldn't click the button "continue". Now I am lost and don't know what to do.

So what do I have to do to get access to my iotas? 

Thanks a lot!

**Comment by begonaalvarezd** [comment]
If you only have a trinity seed or a kdbx seedvault file, you have to migrate your funds to the new network. Unfortunately the current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade (october 4th). Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again.
We will provide a migration tool to resume legacy migrations, but its not finished yet

Will close the issue because Firefly is not intended to provide migration anymore, but it will be a different tool. The Foundation will notify it when the tool is ready 🌷
